sql/clients_last_seen_raw_v1.init.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE
+  clients_last_seen_raw_v1
+PARTITION BY
+  submission_date
+CLUSTER BY
+  sample_id,
+  client_id AS
+SELECT
+  DATE(NULL) AS submission_date,
```

Comment:
apparently `DATE(NULL)` is implicitly casting `NULL` to a timestamp and passing it through [a function for extracting the date part of a timestamp](https://cloud.google.com/bigquery/docs/reference/standard-sql/date_functions#date). I think it would be better in general to use `CAST(NULL AS <TYPE>)` instead, since that works for all types. 
```suggestion
  CAST(NULL AS DATE) AS submission_date,
```